In [1]:
from scipy.stats import skew, kurtosis

### Question 1

In [2]:
monthly_returns = [-2.5, 1.3, -0.8, -1.9, 2.1, 0.5, -1.2, 1.8, -0.5, 2.3,
-0.7, 1.2, -1.5, -0.3, 2.6, 1.1, -1.7, 0.9, -1.4, 0.3,
1.9, -1.1, -0.4, 2.2, -0.9, 1.6, -0.6, -1.3, 2.4, 0.7,
-1.8, 1.5, -0.2, -2.1, 2.8, 0.8, -1.6, 1.4, -0.1, 2.5,
-1.0, 1.7, -0.9, -2.0, 2.7, 0.6, -1.4, 1.1, -0.3, 2.0]

In [3]:
print("Skewness =",skew(monthly_returns))
print("Kurtosis =", kurtosis(monthly_returns))

Skewness = 0.05289580603481756
Kurtosis = -1.2950729066374378


Skewness : A skewness value is 0.052 that nearly equal to 0 that indicates the distribution is nearly symmetric   
Kurtosis : A negative kurtosis value (less than 0) suggests that the distribution is platykurtic, meaning it has lighter tails and a flatter peak compared to a normal distribution

### Question 2

In [5]:
incomes = [2.5, 4.8, 3.2, 2.1, 4.5, 2.9, 2.3, 3.1, 4.2, 3.9,
2.8, 4.1, 2.6, 2.4, 4.7, 3.3, 2.7, 3.0, 4.3, 3.7,
2.2, 3.6, 4.0, 2.7, 3.8, 3.5, 3.2, 4.4, 2.0, 3.4,
3.1, 2.9, 4.6, 3.3, 2.5, 4.9, 2.8, 3.0, 4.2, 3.9,
2.8, 4.1, 2.6, 2.4, 4.7, 3.3, 2.7, 3.0, 4.3, 3.7,
2.2, 3.6, 4.0, 2.7, 3.8, 3.5, 3.2, 4.4, 2.0, 3.4,
3.1, 2.9, 4.6, 3.3, 2.5, 4.9, 2.8, 3.0, 4.2, 3.9,
2.8, 4.1, 2.6, 2.4, 4.7, 3.3, 2.7, 3.0, 4.3, 3.7,
2.2, 3.6, 4.0, 2.7, 3.8, 3.5, 3.2, 4.4, 2.0, 3.4,
3.1, 2.9, 4.6, 3.3, 2.5, 4.9]

In [6]:
print("Skewness =",skew(incomes))
print("Kurtosis =",kurtosis(incomes))

Skewness = 0.22050966381635612
Kurtosis = -0.9452664315355492


Skewness : skewness is close to zero but slightly positive, the distribution is approximately symmetric with a very mild right skew.      
Kurtosis : A negative kurtosis value (less than 0) suggests that the distribution is platykurtic, meaning it has lighter tails and is flatter compared to a normal distribution.

### Question 3

In [7]:
Rating = [4, 5, 3, 4, 4, 3, 2, 5, 4, 3,
5, 4, 2, 3, 4, 5, 3, 4, 5, 3,
4, 3, 2, 4, 5, 3, 4, 5, 4, 3,
3, 4, 5, 2, 3, 4, 4, 3, 5, 4,
3, 4, 5, 4, 2, 3, 4, 5, 3, 4,
5, 4, 3, 4, 5, 3, 4, 5, 4, 3,
3, 4, 5, 2, 3, 4, 4, 3, 5, 4,
3, 4, 5, 4, 2, 3, 4, 5, 3, 4,
5, 4, 3, 4, 5, 3, 4, 5, 4, 3,
3, 4, 5, 2, 3, 4, 4, 3, 5, 4]

In [8]:
print("Skewness =",skew(Rating))
print("Kurtosis =",kurtosis(Rating))

Skewness = -0.20773281879682218
Kurtosis = -0.767917404014467


Skewness : The skewness is slightly negative, indicating that the distribution has a small left (negative) skew.    
Kurtosis : A negative kurtosis value suggests that the distribution is platykurtic, meaning it has thinner tails and is more flat-topped compared to a normal distribution. This implies fewer extreme values (outliers).

### Question 4

In [9]:
house_price = [280, 350, 310, 270, 390, 320, 290, 340, 310, 380,
270, 350, 300, 330, 370, 310, 280, 320, 350, 290,
270, 350, 300, 330, 370, 310, 280, 320, 350, 290,
270, 350, 300, 330, 370, 310, 280, 320, 350, 290,
270, 350, 300, 330, 370, 310, 280, 320, 350, 290,
270, 350, 300, 330, 370, 310, 280, 320, 350, 290,
270, 350, 300, 330, 370, 310, 280, 320, 350, 290,
270, 350, 300, 330, 370, 310, 280, 320, 350, 290,
270, 350, 300, 330, 370, 310, 280, 320, 350, 290,
270, 350, 300, 330, 370, 310, 280, 320, 350, 290]

In [10]:
print("Skewness =",skew(house_price))
print("Kurtosis =",kurtosis(house_price))

Skewness = 0.20606717698636337
Kurtosis = -1.045680283003699


Skewness : The skewness is slightly positive, the distribution is nearly symmetric but has a small rightward (positive) skew.  
Kurtosis : The kurtosis value is negative, which suggests the distribution is platykurtic.

### Question 5

In [13]:
wait_time = [12, 18, 15, 22, 20, 14, 16, 21, 19, 17,
22, 19, 13, 16, 21, 22, 17, 19, 22, 18,
14, 20, 19, 17, 22, 18, 15, 21, 20, 16,
12, 18, 15, 22, 20, 14, 16, 21, 19, 17,
22, 19, 13, 16, 21, 22, 17, 19, 22, 18,
14, 20, 19, 17, 22, 18, 15, 21, 20, 16,
12, 18, 15, 22, 20, 14, 16, 21, 19, 17,
22, 19, 13, 16, 21, 22, 17, 19, 22, 18,
14, 20, 19, 17, 22, 18, 15, 21, 20, 16,
12, 18, 15, 22, 20, 14, 16, 21, 19, 17]

In [15]:
print("Skewness =",skew(wait_time))
print("Kurtosis =",kurtosis(wait_time))

Skewness = -0.32996659307494663
Kurtosis = -0.8969791791415274


Skewness : -0.33 is close to 0, the asymmetry is not very strong—the distribution is almost symmetric but with a slight leftward lean.   
Kurtosis : A negative kurtosis suggests the distribution is platykurtic, meaning it has flatter peaks and thinner tails than a normal distribution